opentelemetry-sdk/src/resource/mod.rs

```diff
@@ -262,13 +258,12 @@ pub trait ResourceDetector {
     ///
     /// If source information to construct a Resource is invalid, for example,
     /// missing required values. an empty Resource should be returned.
-    fn detect(&self, timeout: Duration) -> Resource;
+    fn detect(&self) -> Resource;
```

Comment:
can we keep it obsolete for 0.27.1 release and remove in 0.28?

Reply:
Yup! I can update it tomorrow 👍

Reply:
is there a clean way of marking one argument as deprecated? from what I can tell we wont be able to allow users to remove the usage even if it gets marked as deprecated since we would need to give a new function on the Detector trait

Reply:
I am not able to find a way to do that in Rust! Lets hold on to this and merge 0.27.1 is out (~1 week)